src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs

```diff
@@ -44,14 +44,6 @@ public class GetAzureRmRecoveryServicesBackupContainer : RSBackupVaultCmdletBase
         [ValidateSet("AzureVM", "MARS", "AzureSQL", "AzureStorage")]
         public string BackupManagementType { get; set; }
 
-        /// <summary>
-        /// Friendly name of the container(s) to be fetched. This will be deprecated.
-        /// </summary>
-        [Parameter(Mandatory = false, Position = 3,
-            HelpMessage = ParamHelpMsgs.Container.Name)]
-        [ValidateNotNullOrEmpty]
-        public string Name { get; set; }
```

Comment:
You cannot remove a parameter unless you had previously deprecated it.  Instead, you should map this to 'FriendlyName' and add a deprecation attribute.  See: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md

Reply:
It has been marked depreciated for a while now, you can view the changes in AzureWorkloadProviderHelper.cs.